### ISSUE TITLE
Add additional contact fields

### ISSUE THREAD

**Issue by Emiller88** [nf-core/configs#580]
We might also move from:

```diff
-    config_profile_contact = 'Denis O\'Meally (@drejom) or HPRCC Helpdesk (HPRCC_Help@coh.org)'
+    config_profile_contact = 'Denis O\'Meally'
+    config_profile_contact_github = '@drejom'
+    config_profile_contact_email = "HPRCC_Help@coh.org"

_Originally posted by @Emiller88 in https://github.com/nf-core/configs/issues/577#issuecomment-1776362226_
            
Basically some modules have all of the info, some modules don't. It's fine if they don't want to include an email for example. The real problem is they're not in a consistant format.

So we also need to go through and clean up the CODEOWNERS errors.

**Comment by ewels** [comment]
Agree that this would be good. Originally this was not used for anything except printing to the pipeline stdout for Human eyes to read. But if we're using it programmatically in places then we should split it up 👍🏻 

Downside is more params proliferation in every pipeline. But hopefully not too bad, and should be entirely managed through a (hopefully fairly clean) template update & sync.

**Comment by edmundmiller** [comment]
I feel like that can be in the markdown in the docs.

**Comment by jfy133** [comment]
> Why not something like config_profile_contact_it_email as well to get the email of the IT department?

they might not like being spammed about stuff they don't know about 😬

**Comment by maxulysse** [comment]
I do believe this is done (at least for the github contact from what I can see on the website)
